src/xenia/kernel/xam/xam_notify.cc

```diff
@@ -74,7 +74,10 @@ dword_result_t XNotifyGetNext(dword_t handle, dword_t match_id,
       *param_ptr = param;
     }
   } else {
-    *id_ptr = 0;
+    // id_ptr may be null - "Carcassonne" explicitly passes nullptr in code
+    if (id_ptr) {
```

Comment:
Not handled in the `if (dequeued)` above.

Reply:
I don't know if it should be handled in that case too. Maybe it should, but we have no case of such behaviour

Reply:
If it's a nullable parameter in any case ("maybenull" in SAL terms), null pointer should be passable in all cases, I think.